foundation-ui/navigation/EventTypes.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
I'm surprised `export default` isn't causing any issues here. It was in the past, where we would have to do:
`import Default as EventTypes from '...'`